src/sentry/static/sentry/app/components/searchBar.jsx

```diff
@@ -1,21 +1,18 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import ReactDOM from 'react-dom';
 
 class SearchBar extends React.PureComponent {
   static propTypes = {
     query: PropTypes.string,
     defaultQuery: PropTypes.string,
     onSearch: PropTypes.func,
-    onQueryChange: PropTypes.func,
     placeholder: PropTypes.string,
   };
 
   static defaultProps = {
     defaultQuery: '',
     query: '',
     onSearch: function() {},
-    onQueryChange: function() {},
```

Comment:
What about the component passing `onQueryChange` to this? Was it not being passed?

Reply:
It wasn't passed or used anywhere